src/Controls/PlanarControls.js

```diff
@@ -501,51 +510,81 @@ class PlanarControls extends THREE.EventDispatcher {
             delta = -event.detail;
         }
 
-        const pointUnderCursor = this.getWorldPointAtScreenXY(mousePosition);
+        pointUnderCursor.copy(this.getWorldPointAtScreenXY(mousePosition));
         const newPos = new THREE.Vector3();
 
-        // Zoom IN
-        if (delta > 0) {
-            // decrease the camera field of view if the camera is orthographic
-            if (this.camera.isOrthographicCamera) {
-                startZoom = this.camera.zoom;
-                endZoom = startZoom * (1 + this.zoomInFactor);
-                pointUnderCursor.z = this.camera.position.z;
-            }
-            // target position
-            newPos.lerpVectors(
-                this.camera.position,
-                pointUnderCursor,
-                this.zoomInFactor,
-            );
-            // initiate travel
-            this.initiateTravel(
-                newPos,
-                this.zoomTravelTime,
-                null,
-                false,
-            );
-        // Zoom OUT
-        } else if (delta < 0 && this.maxAltitude > this.camera.position.z) {
-            // increase the camera field of view if the camera is orthographic
+        if (delta > 0 || (delta < 0 && this.maxAltitude > this.camera.position.z)) {
+            const zoomFactor = delta > 0 ? this.zoomInFactor : -1 * this.zoomOutFactor;
+
+            // change the camera field of view if the camera is orthographic
             if (this.camera.isOrthographicCamera) {
+                // switch state to STATE.ZOOM
+                this.state = STATE.ZOOM;
```

Comment:
if this is just for camera ortho, could you rename  to STATE.ORTHO_ZOOM

Reply:
Indeed, it is just for orthographic camera. I shall rename it.